web/src/SidebarTriggerModeToggle.tsx

```diff
@@ -87,16 +87,16 @@ function TriggerModeToggle(props: TriggerModeToggleProps) {
   }
 
   return (
-    <TriggerModeToggleRoot
+    <StyledSidebarTriggerModeToggle
```

Comment:
We've been using the naming convention suffix of `Root` to avoid naming conflicts between React Components and styled-components. But I noticed in the official [docs](https://styled-components.com/docs/basics#define-styled-components-outside-of-the-render-method) that the convention is to use a prefix of `Styled`. 

Since I had to change the name here anyhow, I've added the `Styled` prefix because I think it's more obvious than `Root`.